packages/ui/src/theme.ts

```diff
@@ -0,0 +1,18 @@
+import {axisThemeConfig} from  '@centrifuge/axis-theme'
+import { css } from 'styled-components'
+
+export const theme = {
+  ...axisThemeConfig,
+  dataTable: {
+    ...axisThemeConfig.dataTable,
+    body: {
```

Comment:
This is not part of the feature...just a fixed I sneaked it because we had overlapping texts in the columns